src/workerd/jsg/resource.h

```diff
@@ -679,6 +679,30 @@ struct ResourceTypeBuilder {
     inspectProperties = v8::ObjectTemplate::New(isolate);
     prototype->Set(symbol, inspectProperties, static_cast<v8::PropertyAttribute>(
       v8::PropertyAttribute::ReadOnly | v8::PropertyAttribute::DontEnum));
+
+    auto toStringTagSymbol = v8::Symbol::GetToStringTag(isolate);
+
+    prototype->SetAccessor(toStringTagSymbol, ResourceTypeBuilder::getAndLogToStringTag, &ResourceTypeBuilder::setAndLogToStringTag);
+  }
+
+  static void setAndLogToStringTag(v8::Local<v8::Name> property, v8::Local<v8::Value> value, const v8::PropertyCallbackInfo<void>& info){
+    LOG_ERROR_PERIODICALLY("toStringTag has been set");
+    auto isolate = info.GetIsolate();
```

Comment:
For this it would be simpler just to store the given `value` within a `jsg::Value` member field.

Reply:
I originally tried doing that, but I couldn't get access to the `ResourceTypeBuilder` instance from within this function, since I need to be able to give a pointer to it to SetAccessor. Any suggestions?

Reply:
Also, I think we would have to use the object itself to store the property, since we would want two objets to have separate toStringTags even if they share a single ObjectTemplate.